canonical-web-and-design/maas-ui
# refactor(ui): use ContextualMenu from react-components

## Done

- Replaced the ContextualMenu component with the component from react-components.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

Check the contextual menus look fine for the following pages:
- RSD list take action menu.
- Machine list take action menu.
- Machine list in-table menus.
- Machine list filter accordion.
- KVM list take action menu.
- KVM details take action menu.
- KVM details machine list menu.
- KVM compose form storage location select menu.
- KVM compose form interface menu.


## Fixes

Fixes: canonical-web-and-design/maas-ui#1691
